backend/src/connectors/fileScanning/clamAv.ts

```diff
@@ -73,7 +48,7 @@ export class ClamAvFileScanningConnector extends BaseFileScanningConnector {
         },
       ]
     } catch (error) {
-      return this.scanError(error, file)
+      return this.scanError(`Unable to scan file. ${this.toolName} errored.`, { error, file })
```

Comment:
Maybe reword to `This file could not be scanned due to an error caused by <tool name>`